membership_extension/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Membership extension
+====================
+
+This module extends Odoo's membership management with the following features:
+
+* Membership Category
+* Membership lines editable
+* Do not calculate membership state from invoice status
+* Start date of last membership period
```

Comment:
Explain what a category means: `Membership categories allow to classify memberships by types, allowing a partner to be member or not of the different categories.`